src/main/resources/application.properties

```diff
@@ -13,10 +13,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-
```

Comment:
For some reason, this seems to break the license check. please add the newline here again.